sdk/iOS/src/MSSyncContextInternal.h

```diff
@@ -27,6 +28,8 @@
 
 -(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId completion:(MSSyncBlock)completion;
 
+-(NSOperation *) pullWithQuery:(MSQuery *)query queryId:(NSString *)queryId pullSettings:(MSPullSettings *)pullSettings completion:(MSSyncBlock)completion;
```

Comment:
I meant the method name, not the class name.  pullWithQuery:queryId:pullSettings:completion, I think can be just settings: in this context.